README.md

```diff
@@ -74,119 +75,36 @@ request.clone().serialize_into(&mut encoded)?;
 assert_eq!(AccessTokenRequest::deserialize_from(encoded.as_slice())?, request);
 ```
 
-### Access Tokens
```

Comment:
I've now put this section back in and updated the example.

However (as mentioned before in private chat), I am unsure if the examples we currently provide are actually valid, as we A: send a symmetric key in a signed (i.e. non-encrypted) COSE structure, and B: do not wrap the key in the `cnf` claim as specified in [RFC 9200, section 5](https://datatracker.ietf.org/doc/html/rfc9200#section-5-2.4.1) (which references [RFC 9747, section 3.1](https://datatracker.ietf.org/doc/html/rfc8747#section-3.1)).

Maybe we need to update these examples in general, however, this might also be a candidate for a separate PR, as it is not strictly related to the COSE stuff we add here.

Reply:
Created an issue for this: #21